frontend/src/pages/Resources/components/ResourceSharingForm/ResourceSharingForm.jsx

```diff
@@ -90,355 +139,401 @@ export function ResourceSharingForm(props) {
           : `${style["resource-form"]} ${style["resource-form-light"]} ${style["child2"]}`
       }
     >
-      <div
-        className={
-          dark
-            ? `${style["resource-card"]} ${style["resource-card-dark"]} `
-            : `${style["resource-card"]} ${style["resource-card-light"]}`
-        }
-      >
-        <h3
+      {submitted ? (
+        isLoading ? (
+          <React.Fragment>
+            <div className={style["goodbye-card"]}>
+              <Loader height="25vh" />
+            </div>
+          </React.Fragment>
+        ) : resourceToastStatus === "error" ? (
+          <React.Fragment>
+            <div className={style["goodbye-card"]}>
+              <h1 className={style["card-heading"]}>OOPS !</h1>
+              <div className={style["inside-card"]}>
+                <p style={{ textAlign: "center" }}>
+                  Sorry for the inconvenience caused, our servers are currently
+                  facing some issues. 🔧 <br />
+                  Please try again later!
+                </p>
+              </div>
+            </div>
+          </React.Fragment>
+        ) : (
+          <React.Fragment>
```

Comment:
Instead of `Go Back to Home` can we please show the card similar to other form submission like the contact us one? Please refer https://github.com/HITK-TECH-Community/Community-Website/pull/714 for handling the success and error scenario. The video attached in there shows the card, we need similar card display here as well for the consistency.

Reply:
Hello @Kajol-Kumari, I have updated the changes as per your suggestion. Kindly check

![Screenshot (4595)](https://user-images.githubusercontent.com/55338588/220771287-124a7d3e-6fbc-4cca-b687-2b1425da7bc5.png)